dashboard/app/models/circuit_playground_discount_application.rb

```diff
@@ -0,0 +1,73 @@
+# == Schema Information
+#
+# Table name: circuit_playground_discount_applications
+#
+#  id                                  :integer          not null, primary key
+#  user_id                             :integer          not null
+#  unit_6_intention                    :integer
+#  has_confirmed_school                :boolean          default(FALSE), not null
+#  full_discount                       :boolean
+#  admin_set_status                    :boolean          default(FALSE), not null
+#  signature                           :string(255)
+#  signed_at                           :datetime
+#  circuit_playground_discount_code_id :integer
+#  created_at                          :datetime         not null
+#  updated_at                          :datetime         not null
+#
+# Indexes
+#
+#  index_circuit_playground_applications_on_code_id           (circuit_playground_discount_code_id)
+#  index_circuit_playground_discount_applications_on_user_id  (user_id)
+#
+
+class CircuitPlaygroundDiscountApplication < ApplicationRecord
+  belongs_to :user
```

Comment:
Should this belong to `:studio_person` instead?  At least right now, I don't think we've put anything in place to prevent a user from claiming a code for each of their accounts.

Reply:
That's a good question. My inclination is that `:studio_person` is something we want to use to be able to associate user accounts that belong to the same person, but we may not want to use more broadly.

It is probably worth adding code to make it so that if one user associated with a studio_person already has a discount code, we don't let them request another. Perhaps at that point it does just make sense to belong to a `:studio_person` instead?